packages/web/localizations/en.json

```diff
@@ -379,7 +379,8 @@
     "chartUnavailable": "Chart unavailable",
     "sequenceNumber": "You have too many concurrent transactions! Try resending after your prior tx lands on chain. (Sequence: {sequence})",
     "swapSlippage": "Swap for {coinDenom} failed due to slippage. Try increasing the slippage tolerance.",
-    "invalidAmounts": "Invalid amount(s)"
+    "invalidAmounts": "Invalid amount(s)",
+    "txTimedOutError": "Transaction timed out. Please retry."
```

Comment:
Note: I considered also formatting height but I don't think it would be very useful for users in the current state.

If we do a count to timeout height in the future, it will become more useful